From: Apollon Oikonomopoulos <apollon@skroutz.gr>
Date: Thu, 11 Apr 2013 14:19:44 +0300
Subject: Fix typo in src/haproxy.c

Forwarded: no
Last-Update: 2013-04-11
---
 src/haproxy.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/haproxy.c b/src/haproxy.c
index 7a09e3f..1b55c2f 100644
--- a/src/haproxy.c
+++ b/src/haproxy.c
@@ -716,7 +716,7 @@ void init(int argc, char **argv)
 		      "  is too low on this platform to support maxconn and the number of listeners\n"
 		      "  and servers. You should rebuild haproxy specifying your system using TARGET=\n"
 		      "  in order to support other polling systems (poll, epoll, kqueue) or reduce the\n"
-		      "  global maxconn setting to accomodate the system's limitation. For reference,\n"
+		      "  global maxconn setting to accommodate the system's limitation. For reference,\n"
 		      "  FD_SETSIZE=%d on this system, global.maxconn=%d resulting in a maximum of\n"
 		      "  %d file descriptors. You should thus reduce global.maxconn by %d. Also,\n"
 		      "  check build settings using 'haproxy -vv'.\n\n",
